src/components/resource/ResourceText.jsx

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+import React, {PropTypes} from 'react';
+import HelpTooltip from './HelpTooltip';
+import AdminMenu from './AdminMenu';
+import classNames from 'classnames';
+
+import './ResourceText.scss';
+
+const ResourceText = React.createClass({
+
+  displayName: 'ResourceText',
+
+  propTypes: {
+    className: PropTypes.string,
+    resourceText: PropTypes.shape({
+      text: PropTypes.string,
+      description: PropTypes.string,
+      editUrl: PropTypes.string,
+    }).isRequired,
+    isAdmin: PropTypes.bool,
+  },
+
+  render() {
+    return (
+      <div className={classNames(this.props.className, 'ResourceText')}>
```

Comment:
Extract `className`, `resourceText` and `isAdmin` from `this.props` and apply the remaining props to the enclosing div. This allows parent components to apply event handlers to the `ResourceText` component or add attributes to its HTML tag.